packages/core/src/tracing/index.js

```diff
@@ -160,6 +164,16 @@ exports.preInit = function preInit(preliminaryConfig) {
  * @param {CollectorPIDStore} _processIdentityProvider
  */
 exports.init = function init(_config, downstreamConnection, _processIdentityProvider) {
+  if (hasExperimentalLoaderFlag()) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'Node.js introduced breaking changes in versions 18.19.0 and above, leading to the discontinuation of support ' +
+        `for the experimental loader flag by Instana. The current Node.js version is ${process.version}. ` +
+        "To ensure monitoring by Instana, please use the '--import' flag instead. For more information, " +
```

Comment:
suggestion: use the wording "tracing" such as "Please switch to the --import flag to ensure that tracing works again". (just an example)